sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.ad_activation_performance`
+AS
+WITH client_activation_per_campaign AS (
+  SELECT
+    DATE(submission_timestamp) AS `date`,
```

Comment:
Also - there's a 7 day lag from install to when clients activate. We want to associate a use with their install date, not their activation date